code/__HELPERS/game.dm

```diff
@@ -405,11 +405,11 @@ datum/projectile_data
 	flick_overlay(I, viewing, duration)
 
 
-/proc/window_flash(client/C)
+/proc/window_flash(client/C, ignorepref = FALSE)
 	if(ismob(C))
 		var/mob/M = C
 		if(M.client)
```

Comment:
Given you are checking for `C` in the next line, you can remove this `if(M.client)` check.